website/source/api/secret/pki/index.html.md

```diff
@@ -245,7 +246,15 @@ $ curl \
 ## Set CRL Configuration
 
 This endpoint allows setting the duration for which the generated CRL should be
-marked valid.
+marked valid. If the CRL is disabled, it will return a signed but zero-length
+CRL for any request. If enabled, it will re-build the CRL.
+
+  ~> Note: Disabling the CRL does not affect whether revoked certificates are
+  stored internally. Certificates that have been revoked when a role's
+  certificate storage is enabled will continue to be marked and stored as
```

Comment:
The certificate store shouldn't be a bottleneck because each cert is stored in a different k/v pair in the Consul storage backend - whereas the CRL is stored in a single k/v pair? Is that right @jefferai ?

Reply:
Right.